app/audit/PSPAuthorisationAuditEvent.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package audit
+
+import play.api.libs.json.Json
+
+case class PSPAuthorisationAuditEvent(
+                                         psaId: String,
+                                         pspId: String,
+                                         pstr: String
+                                       ) extends AuditEvent {
+  override def auditType: String = "PensionSchemeAdministratorAuthorisePractitioner"
+
+  override def details: Map[String, String] = {
+    Map(
+      "initiatedIDType" -> "PSAID",
```

Comment:
Shouldn't these strings too have the full expanded version instead of the acronym?

Reply:
You would think so wouldn't you? Emma and I had the same thought when we were coding it. But this is what the TxM team have approved on the TxM page (see https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?pageId=244128549).